bundles/org.openhab.ui/web/src/components/widgets/system/oh-slider.vue

```diff
@@ -10,30 +11,20 @@
 
 <script>
 import mixin from '../widget-mixin'
+import slideMixin from './slide-mixin.vue'
 import { OhSliderDefinition } from '@/assets/definitions/widgets/system'
 
 export default {
-  mixins: [mixin],
+  mixins: [mixin, slideMixin],
   widget: OhSliderDefinition,
   mounted () {
-    delete this.config.value
-
     // f7-range inside of masonry can get rendered faulty, as the masonry changes its breakpoint layout after being rendered
     // re-calculate the range slider after masonry is updated
     setTimeout(() => {
       this.$refs.rangeslider.f7Range.calcSize()
```

Comment:
Not strictly related to this PR, but I just noticed the following error appears quite frequently referencing this line:

```
oh-slider.vue?08ef:24 Uncaught TypeError: Cannot read property 'f7Range' of undefined
    at eval (oh-slider.vue?08ef:24)
```

Perhaps this PR is a good opportunity to solve it as well?

Reply:
Well, added it blindly now. Please test.

Reply:
Looks good, we'll know soon enough if there are major regressions with this new implementation, but I don't any so far!